features/networking/ptp_operator.feature

```diff
@@ -11,3 +11,94 @@ Feature: PTP related scenarios
       | app=linuxptp-daemon |
     And status becomes :running of exactly 1 pods labeled:
       | name=ptp-operator |
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-25738
+  @admin
+  Scenario: Linuxptp run as slave can sync to master
```

Comment:
@huiran0826 not a deal breaker, but please avoid using terms `slave` and `master` to have a more appropriate nomenclature.  I think replacing with `slave -> secordary`, `master -> primary` would work.

Reply:
`master` and `slave` are specific terms used in at least the 2002 version of the IEEE 1588 standard.  We should not deviate from the standard unless/until it is updated.  I don't have access to IEEE 1588-2008 but it doesn't look like terms have been changed.

If we don't conform to the standard we shouldn't claim to support it as a feature.

Reply:
@rbbratta Thanks for the explanation.